helm/hpcc/templates/_helpers.tpl

```diff
@@ -2058,3 +2058,27 @@ spec:
 {{- end -}}
 {{- end -}}
 
+{{- define "hpcc.generateWarnings" -}}
+{{/*
+Generate a list of warnings
+*/}}
+{{ $warnings := include "hpcc.getWarnings" .root | fromJsonArray }}
+{{ if $warnings }}
+warnings:
+ {{ range $warning := $warnings }}
+  - msg: {{ $warning.msg | quote }}
```

Comment:
indent: indent looks wrong.  Will this cause problems processing the file?

Reply:
Strangely, it did work.  But I'll fix the format.